services/softlayer_virtual_guest_test.go

```diff
@@ -37,6 +38,9 @@ var _ = Describe("SoftLayer_Virtual_Guest_Service", func() {
 		fakeClient = slclientfakes.NewFakeSoftLayerClient(username, apiKey)
 		Expect(fakeClient).ToNot(BeNil())
 
+		// Use mock for product package service (which provides mock pricing items for "cpu", "ram" and "network speed")
```

Comment:
no need for this comment. Code is clear. Anyone can read code for Mock and see what features it exposes.